contents/handbook/people/spending-money.md

```diff
@@ -12,7 +12,7 @@ Please spend money in a way that you think is in the best interests of the compa
 
 If it's a trivial expense, just buy it. We provide you with a company card with a $1,000/month spending limit for this reason. We use Brex for everyone, and also provide UK team members with a Revolut card. 
 
-If you live in the UK, you should use your Revolut card for UK-specific spending (i.e. ordering from UK sites), and Brex for everything else. This is for UK accounts-reporting reasons, as we have a UK subsidiary. 
+You should use your Revolut card for UK-specific spending (i.e. ordering from UK sites, hotels/food/drink/... in the UK), and Brex for everything else. This is for UK accounts-reporting reasons, as we have a UK subsidiary.
```

Comment:
Should people living in America get Revolut cards too? Not sure if it's worth it, but at least @fuziontech have gone twice to London now for PostHog business.

Reply:
I'd say not worth it - you can't claim VAT back on travel (flights, trains etc.) so the only benefit is that we can claim back 20% on food expenses, which will be minimal. And if you are doing a large group meal, a UK person should just pay the bill.

Reply:
Then maybe update this in the wording here?